strapi/strapi
# Fix GraphQL enums are genereated as String type

Signed-off-by: harimkims <harimkims@gmail.com>



### What does it do?

- I fixed GraphQL enums are generated as String type - (Input type either)
- I changed the order of if statements that distinguish type. Because Enum should be treated as enum before being treated as a scalar.

### Why is it needed?

- In order to generate correct type

### How to test it?

1. Create content type has enum field(attribute)
2. Go to `localhost:1337/graphql`
3. Click 'docs' at right of the window
4. See if they are generated as enum type in Query and Mutation.

### Related issue(s)/PR(s)

Closes #11993
